themes/cerberus/index.txt

```diff
@@ -1,31 +1,31 @@
-<%- title %>
-
+<%- title %><br />
+<br />
 <% if (locals.intro) { %>
 <% intro.forEach(function (introItem) { -%>
-<%- introItem %>
+<%- introItem %><br />
 <% }) -%>
+<br />
 <% } %>
-
 <% if (locals.dictionary) { %>
 <% for (item in dictionary) { -%>
-<%- item.charAt(0).toUpperCase() + item.slice(1) %>: <%- dictionary[item] %>
+<%- item.charAt(0).toUpperCase() + item.slice(1) %>: <%- dictionary[item] %><br />
 <% } -%>
+<br />
 <% } %>
-
 <% if (locals.action) { %>
 <% action.forEach(function (actionItem) { -%>
-<%- actionItem.instructions %>
-<%- actionItem.button.link %>
+<%- actionItem.instructions %><br />
+<%- actionItem.button.link %><br />
 <% }) -%>
+<br />
```

Comment:
I think this `<br />` should be moved inside the `action.foreach` loop so that the output appears like this:

```
To get started with Mailgen, please click here:
https://mailgen.js/confirm?s=d9729feb74992cc3482b350163a1a010

To get started with Mailgen, please click here:
https://mailgen.js/confirm?s=d9729feb74992cc3482b350163a1a010
```

And not like this:
```
To get started with Mailgen, please click here:
https://mailgen.js/confirm?s=d9729feb74992cc3482b350163a1a010
To get started with Mailgen, please click here:
https://mailgen.js/confirm?s=d9729feb74992cc3482b350163a1a010

```